pkgs/by-name/sw/swayosd/package.nix

```
@@ -12,25 +12,26 @@
 , meson
 , ninja
 , rustc
+, sassc
 , stdenv
 , udev
 }:
 
 stdenv.mkDerivation rec {
   pname = "swayosd";
-  version = "unstable-2023-09-26";
+  version = "unstable-2024-04-15";
```

```suggestion
  version = "0-unstable-2024-04-15";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

---

Done!

---

you will also need to adjust commit message and PR title